torch/testing/_internal/common_optimizers.py

```diff
@@ -314,14 +317,12 @@ def optim_inputs_func_adadelta(device, dtype=None):
         OptimizerInput(
             params=None,
             kwargs={"weight_decay": 0.1, "maximize": True},
-            desc="maximize",
+            desc="maximize, weight_decay",
         ),
         OptimizerInput(
             params=None, kwargs={"rho": 0.95, "weight_decay": 0.9}, desc="rho"
         ),
-        OptimizerInput(
-            params=None, kwargs={"maximize": True}, desc="maximize, no weight_decay"
-        ),
+        OptimizerInput(params=None, kwargs={"maximize": True}, desc="maximize"),
```

Comment:
nit: can this be inserted above the other config so it goes from simplest to most complex?

Reply:
same for the below